src/gui/shader.cpp

```diff
@@ -37,13 +38,17 @@ load_shaders(
         vertex_shader_stream.close();
     } else {
         LOG_ERROR(
-            logger, "Cannot open {}. Are you in the right directory?", vertex_file_path
+            logging::opengl_logger, "Cannot open {}. Are you in the right directory?",
+            vertex_file_path.lexically_normal().string()
         );
         return 0;
     }
 
     // Read the Fragment Shader code from the file
-    LOG_INFO(logger, "Loading fragment shader from {}", vertex_file_path);
+    LOG_INFO(
+        logging::opengl_logger, "Loading fragment shader from {}",
+        vertex_file_path.lexically_normal().string()
```

Comment:
Already garunteed to be lexically normal in `utils/files.hpp`

Also don't need the cast to string.